### PR TITLE
Directory tests

### PR DESCRIPTION
**Pull request by Kishan-Dhakan**
Re-open PR for dir-tests to investigate fails.

### CODE REVIEW

**Comment by stewartie4** [comment]
these weren't failing initially - strange that they're failing now unless something has suddenly broken?

**Comment by Kishan-Dhakan** [comment]
@cnlangzi please don't merge this upon passing yet, I had moved some cases to `Broken` scenarios so I need to move them back, Thanks

**Comment by cnlangzi** [comment]
@Kishan-Dhakan other related issues are raised on https://github.com/0chain/blobber/issues/590 and https://github.com/0chain/blobber/issues/592

**Comment by stewartie4** [comment]
this looks ok to review again @Kishan-Dhakan @mohsenno1 right?

**Comment by Kishan-Dhakan** [comment]
@stewartie4 yep, OOMKilled issue is solved but almost every case is a FIXME. If it's not an issue, this should be ready to merge

**Comment by stewartie4** [comment]
Maybe we should move these files to broken scenarios so we can better keep track on them

**Comment by stewartie4** [comment]
where are we with this @mohsenno1?

**Comment by Kishan-Dhakan** [comment]
@stewartie4 I fixed the tests but directory operations are not working at the moment. I think this will be done after mainnet, see: https://0chain.slack.com/archives/C035RK9PTU0/p1652452527605049?thread_ts=1652443141.811229&cid=C035RK9PTU0.

cc @cnlangzi please let us know whether coverage for directory operations is needed at the moment

File: tests/cli_tests/zboxcli_dir_copy_test.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+func TestCopyDir(t *testing.T) {
+	t.Parallel()
+
+	t.Run("copy root directory", func(t *testing.T) {
+		t.Parallel()
+
+		allocationID := setupAllocation(t, configPath)
+		defer createAllocationTestTeardown(t, allocationID)
+
+		dirname := "/rootdir"
```
Comment (lpoli) [inline tests/cli_tests/zboxcli_dir_copy_test.go:22]: `/rootdir` is not root directory. `/` is.

**Comment by lpoli** [comment]
@stewartie4 @Kishan-Dhakan 
Lets not merge anything by tagging it as `FIXME` or as `Broken Test`. This will delay bug fixing.

Its better to raise issue and link respective system test PR and assign to somebody in the blobber/gosdk team.

**Comment by cnlangzi** [comment]
> https://0chain.slack.com/archives/C035RK9PTU0/p1652452527605049?thread_ts=1652443141.811229&cid=C035RK9PTU0

Yes. I would suggest to implement directory features/commands after mainnet. Because I think it is not good to implement it as a normal synchronous restful api

**Comment by stewartie4** [comment]
@cnlangzi agreed, multiple files in an allocation would be very taxing on server side 
@lpoli agreed

**Comment by Kishan-Dhakan** [comment]
@lpoli @cnlangzi @stewartie4 great, we're on the same page.

**Comment by Kishan-Dhakan** [comment]
Closing this, needs to be reopened post-mainnet.

**Pull request closed without merging**